### PR TITLE
COM-477 Getting Started + Storybook improvements + IDEA folder

### PR DESCRIPTION
**Pull request by gvillo**
**What is the ticket / issue associated with this change?** (Link to the ticket or issue)

- Adds useful addons for storybook
- Improves gitignore file
- Adds IDEA folder
- Add script to run the app at root level
- Improves READMEs with getting started  
- Migrated turbo config to `turbo.json`

### CODE REVIEW

**Review by tkanzakic — DISMISSED** [review]
You have to update the Dockerfile too to use the new commando to build the storybook

**Comment by gvillo** [comment]
> You have to update the Dockerfile too to use the new commando to build the storybook

thank you, fixed it!

File: projects/ui-storybook/.storybook/main.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     '../../../packages/signalwire/src/**/*.stories.tsx',
   ],
   addons: [
+    '@storybook/addon-a11y',
```
Comment (gvillo) [inline projects/ui-storybook/.storybook/main.ts:13]: really good addon to add Accessibility tab to storybook

File: package.json [hunk 2]
```diff
@@ -23,45 +23,6 @@
     "packages/*",
     "projects/*"
   ],
-  "turbo": {
```
Comment (gvillo) [inline package.json:26]: this way of adding turbo was deprecated, so I moved to a turbo.json file

File: packages/components/src/components/dataDisplay/NumberedList/NumberedList.tsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ const NumberedListItem: FC<NumberedListItemProps> = ({
     <Box>
       {Children.map(children, (child) => {
         if (isValidElement(child) && child.type === NumberedList) {
-          return cloneElement(child, { number, paddingTop: '0.75rem' });
+            return cloneElement(child, { number, paddingTop: '0.75rem' } as NumberedListProps);
```
Comment (tkanzakic) [inline packages/components/src/components/dataDisplay/NumberedList/NumberedList.tsx:36]: Is this typecast necessary?, also there is an extra indentation that does not follow the rest of the repo

File: packages/components/README.md [hunk 4]
```diff
@@ -4,6 +4,13 @@
 
 The composer component library is an _opinionated_ wrapper on top of [chakra-ui](https://chakra-ui.com/) that allows defining a branded theme instead of a generic one designed to work with the composer ecosystem.
 
+## Running the app
```
Comment (tkanzakic) [inline packages/components/README.md:7]: This is no longer valid, the storyboook is launched from the root of the rep

File: packages/blocks/src/AccordionGallery/components/AccordionGalleryAccordion.tsx [hunk 5]
```diff
@@ -8,7 +8,7 @@ export const AccordionGalleryAccordion: FC<AccordionProps> = ({ children, ...res
       if (isValidElement(child)) {
         switch (child.type) {
           case AccordionGalleryItem:
-            return cloneElement(child, { index });
+            return cloneElement(child, { index } as AccordionProps);
```
Comment (tkanzakic) [inline packages/blocks/src/AccordionGallery/components/AccordionGalleryAccordion.tsx:11]: Is this typecast necessary?

File: README.md [hunk 6]
```diff
@@ -2,7 +2,26 @@
...
 
-## Changesets
+This is a monorepo that supports components, analytics, 3rd party integrations (such as contentful), payment methods, messaging and much more.
+
+## Getting Started
+
+You can find all the implementations `packages` folder. Every package has his own readme.
+
+One best way to start working on this project is to run [components package](https://github.com/cmpsr/composer/blob/master/packages/components/README.md).
+You can do it from the root of this folder by running `yarn run components:storybook`, please be asure to run `yarn install` first on packages.
```
Comment (tkanzakic) [inline README.md:12]: This is no longer true, the storybook is launched from the root of the repo

File: README.md [hunk 7]
```diff
@@ -2,7 +2,26 @@
 
 # Composer
 
-## Changesets
+This is a monorepo that supports components, analytics, 3rd party integrations (such as contentful), payment methods, messaging and much more.
+
+## Getting Started
+
+You can find all the implementations `packages` folder. Every package has his own readme.
```
Comment (tkanzakic) [inline README.md:9]:
```suggestion
You can find all the implementations in the `packages` folder. Every package has his own readme.
```

File: README.md [hunk 8]
```diff
@@ -2,7 +2,26 @@
 
 # Composer
 
-## Changesets
+This is a monorepo that supports components, analytics, 3rd party integrations (such as contentful), payment methods, messaging and much more.
```
Comment (tkanzakic) [inline README.md:5]: We do not support payment methods 😉

Comment (tkanzakic) [reply README.md:5]: And we have a better definition of composer _somewhere_, I will search for it and share it with you

File: README.md [hunk 9]
```diff
@@ -2,7 +2,26 @@
...
+
+## How to contribute
+
+### Adding new UI components to the library
+
+For adding new components follow the [implementation guidelines](https://github.com/cmpsr/composer/blob/master/packages/components/README.md).
+
+### Adding new package to this repo
+
+Add the following package under `packages` folder and be sure you add the README file, scripts on root package.json, so everyone can understand how to use it correctly. Don't forget to push vscode and idea run configurations too!
```
Comment (tkanzakic) [inline README.md:22]: We should reword this statement

**Review by tkanzakic — CHANGES_REQUESTED** [review]

**Pull request closed without merging**